crates/nu-command/src/formats/to/raw.rs

```diff
@@ -0,0 +1,97 @@
+use nu_engine::command_prelude::*;
+use nu_protocol::RawStream;
+
+#[derive(Clone)]
+pub struct ToRaw;
+
+impl Command for ToRaw {
+    fn name(&self) -> &str {
+        "to raw"
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build(self.name())
+            .input_output_types(vec![
+                (Type::String, Type::Any),
+                (Type::Binary, Type::Any),
+                (Type::List(Type::Any.into()), Type::Any),
+            ])
+            .category(Category::Formats)
+    }
+
+    fn usage(&self) -> &str {
+        "Convert a stream to raw input."
+    }
+
+    fn extra_usage(&self) -> &str {
+        r#"
+Renders input of string or binary value(s) as raw input data, the same as if it
+were received as the output of an external command.
+
+This is helpful when trying to print binary data as the output of a script, and
+it also preserves the streaming characteristics of the input, unlike `bytes
+collect`, so it can be used in situations where collecting all of the data at
+once at first would be undesirable.
+"#
+        .trim()
+    }
+
+    fn search_terms(&self) -> Vec<&str> {
+        vec!["binary", "bytes"]
+    }
+
+    fn examples(&self) -> Vec<Example> {
+        vec![
+            Example {
+                example: "0x[aa bb cc dd] | to raw | print",
```

Comment:
I don't understand the point of these `| print` commands. I tested your `to raw` command and it seems to implicitly print like it's supposed to without the `print` command. What am I missing?

Reply:
That's true... But there isn't that much more you can do with raw input, since it often just gets collected or streamed as string/binary implicitly. There isn't a way to explicitly print binary from a script without this though, so I figured it would make sense to include that in the example. Without a `print` or `save` it just sort of feels like... "and then what?"

Reply:
The output was identical on my mac so I thought it was just confusing, "to raw needs a print to work?" was what kept running through my mind. I'm not sure it's important to reinforce that it needs print when it doesn't. However, we could have one example with a more verbose description that explains the use. Are there other commands that can manipulate this data like maybe decode or parse? I switched back to windows and haven't tested it here yet.

Reply:
I agree, it doesn't need to be in every example.

There definitely are other commands that could do useful things with raw input even if they don't require it. `parse` sounds like a very good one to demo

In some ways this can actually act as kind of like a streaming `str join` which makes me wonder if this is really the right name for it, though it logically makes sense and we have `to text` already